WordPress/twentytwentyfive
# Add pattern: Contact - Location and directions link

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/62

**Screenshots**

https://github.com/user-attachments/assets/477bde7c-a3cf-444f-9af1-43db77b14a73

**Testing Instructions**

1. Create a page.
2. Add the pattern.
3. Confirm that it's looking like the designs (spacing/colors/image/etc).
